apps/ensrainbow/src/ingest.ts

```diff
@@ -24,6 +24,11 @@ async function loadEnsNamesToLevelDB(): Promise<void> {
     keyEncoding: "binary",
   });
 
+  // Clear existing database before starting
+  console.log("Clearing existing database...");
+  await db.clear();
+  console.log("Database cleared.");
```

Comment:
It may take a lot of time to clear this way. The alternative is to to delete db directory.
The other way os to assert that the db is empty.